ovs/lib/generic.py

```diff
@@ -73,120 +73,32 @@ def snapshot_all_vdisks():
     @staticmethod
     @ovs_task(name='ovs.generic.delete_snapshots', schedule=Schedule(minute='1', hour='2'), ensure_single_info={'mode': 'DEFAULT'})
     def delete_snapshots(timestamp=None):
+        # type: (float) -> Dict[str, List[str]]
         """
         Delete snapshots & scrubbing policy
 
-        Implemented delete snapshot policy:
+        Implemented default delete snapshot policy:
         < 1d | 1d bucket | 1 | best of bucket   | 1d
         < 1w | 1d bucket | 6 | oldest of bucket | 7d = 1w
         < 1m | 1w bucket | 3 | oldest of bucket | 4w = 1m
         > 1m | delete
-
-        :param timestamp: Timestamp to determine whether snapshots should be kept or not, if none provided, current time will be used
+        The configured policy can differ from this one.
+        :param timestamp: Timestamp to determine whether snapshots should be kept or not,
+        if none provided, the current timestamp - 1 day is used. Used in unittesting only!
+        The scheduled task will not remove snapshots of the current day this way!
         :type timestamp: float
-
-        :return: None
+        :return: Dict with vdisk guid as key, deleted snapshot ids as value
+        :rtype: dict
         """
-        GenericController._logger.info('Delete snapshots started')
+        if os.environ.get('RUNNING_UNITTESTS') == 'False':
```

Comment:
We have a function for this in develop now

Reply:
This isn't develop